fastly/resource_fastly_service_v1.go

```diff
@@ -3526,6 +3535,11 @@ func resourceServiceV1Read(d *schema.ResourceData, meta interface{}) error {
 			return err
 		}
 
+		// Refresh Scalyr
+		if err := readScalyr(conn, d, s); err != nil {
+			return err
+		}
```

Comment:
I'm interested, your other PR's for new endpoints haven't included this read. If correct we should fix in the others.

Reply:
Absolutely! I think they should (and I will confirm) that they do.